packages/components/src/Calendar/CalendarPopover/CalendarPopover.tsx

```diff
@@ -1,14 +1,20 @@
 import React, { HTMLAttributes, useState } from "react"
-import { Options } from "@popperjs/core"
+import {
+  autoUpdate,
+  offset,
+  useFloating,
+  UseFloatingOptions,
+  size,
+  autoPlacement,
+} from "@floating-ui/react-dom"
 import classnames from "classnames"
-import { usePopper } from "react-popper"
 import { OverrideClassName } from "~types/OverrideClassName"
 import styles from "./CalendarPopover.module.scss"
 
 export type CalendarPopoverProps = {
   children: React.ReactNode
   referenceElement: HTMLElement | null
-  popperOptions?: Partial<Options>
+  floatingOptions?: Partial<UseFloatingOptions>
```

Comment:
This would be a breaking change (therefore changeset would need to change) - are we sure no one uses this component outside our repo?

Reply:
Yep no repo's outside of Kaizen use it and we only use it for DatePicker and DateRangePicker where we don't expose this